### PR TITLE
OBPIH-3824 Fix setting date invoiced on prepayment invoices

### PR DESCRIPTION
**Pull request by DominikaHatala**

### CODE REVIEW

File: grails-app/services/org/pih/warehouse/invoice/InvoiceService.groovy [hunk 1]
```diff
@@ -254,7 +255,7 @@ class InvoiceService {
         invoice.description = order.description
         invoice.partyFrom = order.destinationParty
         invoice.party = order.origin.organization
-        invoice.dateInvoiced = new Date()
+        invoice.dateInvoiced = LocalDate.now().toDate()
```
Comment (awalkowiak) [inline grails-app/services/org/pih/warehouse/invoice/InvoiceService.groovy:258]: What is wrong with the previous version?

Comment (DominikaHatala) [reply grails-app/services/org/pih/warehouse/invoice/InvoiceService.groovy:258]: It set the date with time, filter is searching for the date with no time set so it didn't match.

Comment (jmiranda) [reply grails-app/services/org/pih/warehouse/invoice/InvoiceService.groovy:258]: I think we're ok with keeping the time.

**Review by jmiranda — COMMENTED** [review]
LGTM. Unless the requirements specify that invoiced date should only include date, then we should allow the time component.

**Pull request merged**